build.sbt

```diff
@@ -41,7 +41,7 @@ lazy val root =
   project
     .in(file("."))
     .settings(publish / skip := true)
-    .aggregate(opentracing, opentelemetry, opencensus, opentracingExample, opentelemetryExample)
+    .aggregate(opentracing, opentelemetry, opencensus)
```

Comment:
Note that this will exclude them completely. Make sure to create compilation steps for these projects and run them separately, e.g.:

```
// in build.sbt
addCommandAlias("compileExamples", "opentracingExample/compile;opentelemetryExample/compile")

// in ci config
name: Compile examples
if: ${{ startsWith(matrix.scala, '2.') }}
run: ./sbt ++${{ matrix.scala }}! compileExamples
```

Reply:
@KadekM :arrow_up: :smile:

Reply:
thanks for writing the YAML part, I was afraid to touch it :D